server/views/workflows.py

```diff
@@ -194,7 +194,8 @@ def render_workflow(request: HttpRequest, workflow: Workflow):
             #
```

Comment:
Comment above says "edge cases or maybe bugs" but this code path is normal operation for cron, yes?

Reply:
Ooh good catch. I'll add that to the comment.